Lib/defconQt/metricsWindow.py

```diff
@@ -77,12 +76,13 @@ def __init__(self, font, string=None, pointSize=defaultPointSize,
         self.setWindowTitle(font=self.font)
 
     def setupFileMenu(self):
-        fileMenu = QMenu("&File", self)
-        fileMenu.addAction("&Save…", self.save, QKeySequence.Save)
-        fileMenu.addAction("E&xit", self.close, QKeySequence.Quit)
+        fileMenu = QMenu(self.tr("&File"), self)
+        fileMenu.addAction(self.tr("&Save…"), self.save, QKeySequence.Save)
+        fileMenu.addAction(self.tr("&Close"), self.close, QKeySequence.Quit)
         self.menuBar().addMenu(fileMenu)
 
     def setWindowTitle(self, title="Metrics Window", font=None):
+        title = self.tr("Metrics Window")
```

Comment:
Same here with default parameter, switch it to None and change logic acordingly.